ui/src/app/features/model-table/model-table.component.html

```diff
@@ -66,6 +66,10 @@
           <button mat-menu-item (click)="test.emit(element)">
             <span>{{ 'models.select_menu.test' | translate }}</span>
           </button>
+          <!-- NOTE: this functionality is temporarily disabled -->
```

Comment:
why is it disabled?

Reply:
Because it looks like that on a backed side this logic is still not been implemented